test/JDBC/input/ISC-Check-Constraints-vu-prepare.sql

```diff
@@ -29,8 +29,8 @@ create table test_tsql_cast(
 go
 
 create table test_tsql_collate(
-	c_varchar varchar check(c_varchar <> cast('sflkjasdlkfjf' as varchar(12)) COLLATE bbf_unicode_cp1_ci_as),
-	c_char char check(c_char <> cast('sflkjasdlkfjf' as char(7)) COLLATE bbf_unicode_cp1_ci_as),
+	c_varchar varchar check(c_varchar <> cast('sflkjasdlkfjf' as varchar(12)) COLLATE latin1_general_ci_as),
```

Comment:
Existing test cases are okay but let's add test cases for collation outside of this translation table, for example, japanese_ci_as

Reply:
Done.